spec/models/pubmed_source_record_spec.rb

```diff
@@ -145,6 +145,18 @@ def check_author_hash(author)
     end
   end
 
+  describe '.pubmed_update' do
+    it 'updates the :source_data field' do
+      record = described_class.get_pubmed_record_from_pubmed(pmid_created_1999)
+      original_source_data = record.source_data
+      expect(record.source_data).not_to be_empty
+      record.source_data = ''
+      expect(record.source_data).to be_empty
+      expect(record.pubmed_update).to be true
+      expect(record.source_data).to eq original_source_data
```

Comment:
I can't get mocking to work (getting an ```ActiveRecord::StaleObjectError``` error from the database on the "update_attributes!" call in the model).  I am reluctant to sink too much time into testing this new method.